sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala

```diff
@@ -52,6 +52,8 @@ class AdaptiveQueryExecSuite
 
   import testImplicits._
 
+  override protected def sparkConf = super.sparkConf.set("spark.io.compression.codec", "lz4")
```

Comment:
Should this be define more like a method with braces, etc or is that how similar code does it?

Reply:
Sorry for late reply. 

For this one, we can use this form simply like the following.
```
$ git grep 'override protected def sparkConf' | grep super
sql/core/src/test/scala/org/apache/spark/sql/AggregateHashMapSuite.scala:  override protected def sparkConf: SparkConf = super.sparkConf
sql/core/src/test/scala/org/apache/spark/sql/AggregateHashMapSuite.scala:  override protected def sparkConf: SparkConf = super.sparkConf
sql/core/src/test/scala/org/apache/spark/sql/AggregateHashMapSuite.scala:  override protected def sparkConf: SparkConf = super.sparkConf
sql/core/src/test/scala/org/apache/spark/sql/connector/FileDataSourceV2FallBackSuite.scala:  override protected def sparkConf: SparkConf = super.sparkConf.set(SQLConf.USE_V1_SOURCE_LIST, "")
sql/core/src/test/scala/org/apache/spark/sql/execution/DataSourceScanExecRedactionSuite.scala:  override protected def sparkConf: SparkConf = super.sparkConf
sql/core/src/test/scala/org/apache/spark/sql/execution/DataSourceScanExecRedactionSuite.scala:  override protected def sparkConf: SparkConf = super.sparkConf
sql/core/src/test/scala/org/apache/spark/sql/execution/DataSourceScanExecRedactionSuite.scala:  override protected def sparkConf: SparkConf = super.sparkConf
sql/core/src/test/scala/org/apache/spark/sql/execution/adaptive/AdaptiveQueryExecSuite.scala:  override protected def sparkConf = super.sparkConf.set("spark.io.compression.codec", "lz4")
sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/AlterTableRecoverPartitionsSuite.scala:  override protected def sparkConf = super.sparkConf
sql/core/src/test/scala/org/apache/spark/sql/execution/command/v1/AlterTableRecoverPartitionsSuite.scala:  override protected def sparkConf = super.sparkConf
sql/core/src/test/scala/org/apache/spark/sql/execution/datasources/orc/OrcPartitionDiscoverySuite.scala:  override protected def sparkConf: SparkConf = super.sparkConf.set(SQLConf.USE_V1_SOURCE_LIST, "")
sql/core/src/test/scala/org/apache/spark/sql/streaming/StreamSuite.scala:  override protected def sparkConf: SparkConf = super.sparkConf
```

Or, like the following~
```
  override protected def sparkConf: SparkConf =
    super
      .sparkConf
      .set(SQLConf.USE_V1_SOURCE_LIST, "parquet")
```